auth0/src/test/java/com/auth0/android/authentication/storage/SecureCredentialsManagerTest.kt

```diff
@@ -314,25 +314,14 @@ public class SecureCredentialsManagerTest {
         exception.expect(CredentialsManagerException::class.java)
         exception.expectMessage("Credentials must have a valid date of expiration and a valid access_token or id_token value.")
         val credentials: Credentials =
-            CredentialsMock(null, null, "type", "refreshToken", Date(), "scope")
-        manager.saveCredentials(credentials)
-    }
-
-    @Test
-    public fun shouldThrowOnSaveIfCredentialsDoesNotHaveExpiresAt() {
-        exception.expect(CredentialsManagerException::class.java)
-        exception.expectMessage("Credentials must have a valid date of expiration and a valid access_token or id_token value.")
-        val date: Date? = null
-        val credentials: Credentials =
-            CredentialsMock("idToken", "accessToken", "type", "refreshToken", date, "scope")
-        prepareJwtDecoderMock(Date())
+            CredentialsMock("", "", "type", "refreshToken", Date(), "scope")
         manager.saveCredentials(credentials)
     }
 
     @Test
     public fun shouldNotThrowOnSaveIfCredentialsHaveAccessTokenAndExpiresIn() {
         val credentials: Credentials =
-            CredentialsMock(null, "accessToken", "type", "refreshToken", Date(), "scope")
+            CredentialsMock("", "accessToken", "type", "refreshToken", Date(), "scope")
```

Comment:
this case will never happen again, as Credentials will always have these properties passed as empty here. I'd remove this test case.